api/src/main/data/xlit_scheme-utf8.xml

```diff
@@ -384,6 +397,20 @@
 <doc>ISO 11940-2 is an ISO standard for the romanization of the Thai alphabet almost identical to RTGS, published in 2007. https://en.wikipedia.org/wiki/ISO_11940-2</doc>
 </scheme>
 <scheme>
+<numeric>315</numeric>
+<name>mlcts</name>
+<presentationName>MLCTS</presentationName>
+<description>Myanmar Language Commission Transcription System</description>
+<doc>The Myanmar Language Commission Transcription System (1980), also known as the MLC Transcription System (MLCTS), is a transliteration system for rendering Burmese in the Latin alphabet. https://en.wikipedia.org/wiki/MLC_Transcription_System</doc>
```

Comment:
is the description necessary for the doc tag?

Reply:
this is the same format as the Thai xlit scheme above.